### PR TITLE
OSDOCS-3532: Updating example output to 1.24

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3532

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3532/welcome/ (throughout doc set)

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please review these updates that are just bumping the kube version from 1.23 to 1.24 in our example output?

**Review by sheriff-rh — APPROVED** [review]
This looks sane to me. Good work!

File: modules/nodes-nodes-viewing-listing.adoc [hunk 1]
```diff
@@ -156,8 +156,8 @@ System Info:    <9>
  Operating System:                        linux
  Architecture:                            amd64
  Container Runtime Version:               cri-o://1.16.0-0.6.dev.rhaos4.3.git9ad059b.el8-rc2
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/nodes-nodes-viewing-listing.adoc:158]: Good notice, updated!

**Pull request merged**